### PR TITLE
define k3s related aws resources

### LINKED ISSUE #65: terraforming the lightsail resources

### PR DESCRIPTION
**Pull request by bellatoris**
fix #65

### CODE REVIEW

**Comment by bellatoris** [comment]
Current dependencies on/for this PR:
* main
  * **PR #55** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/55" target="_blank">Graphite</a> 
    * **PR #56** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/56" target="_blank">Graphite</a> 
      * **PR #57** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/57" target="_blank">Graphite</a> 
        * **PR #59** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/59" target="_blank">Graphite</a> 
          * **PR #60** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/60" target="_blank">Graphite</a> 
            * **PR #76** <a href="https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/76" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/76?utm_source=stack-comment).

File: infra/terraform/lightsail/secret.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+k3s:
```
Comment (Zeniuus) [inline infra/terraform/lightsail/secret.yaml:1]: sops 좋아보이더라구요
git에 커밋하는 게 행복

File: infra/terraform/lightsail/secret.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    token: ENC[AES256_GCM,data:mfv0bzc22LMQY+QwqhfzgsVAkualJqXvdTfxhyDPguA=,iv:qq+HqLsl76PtW/fmagGlV78Sfg113ZIEZSNNBnUfH+A=,tag:MoatVAJnvSS1Wu2gl5zIrA==,type:str]
+db:
+    password: ENC[AES256_GCM,data:WxYZW1emmoCjlTIkBfixLQ2FufwqNckoalqj8ICm9zvHOfyQm3JYJA==,iv:d8/Jo/NG6/LmZdaZPMQ3lDsbfHoxcLGdoKOtWLU8Whc=,tag:4NUVFgCJyIjWve7ayIYcwA==,type:str]
+sops:
+    kms:
+        - arn: arn:aws:kms:ap-northeast-2:833004893731:alias/sops
+          created_at: "2022-10-21T17:12:31Z"
+          enc: AQICAHj7gmpDUHbiugk8njCuq/LhoXhddxY3AclpiLREvOHpwgHMbtNDU11BXucH/oAaeE/kAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM6Oc6ukpRCHQjnr+9AgEQgDttjvGJ8BiB2B3gT/yQ/8dhDCcRZkbtyXUUQIUKpyefjaGzS8RnSqmJ6OgI1lB8mHk4FPf2Rn+GdTyTcA==
+          aws_profile: ""
```
Comment (Zeniuus) [inline infra/terraform/lightsail/secret.yaml:10]: 궁금한 게, 테라폼 리소스 생성 순서가 kms -> sops secret 파일 -> sops secret 파일에 의존하는 것들이 되어야 하나요?

File: infra/terraform/lightsail/lightsail_rdb.tf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+resource "aws_lightsail_database" "scc_db" {
+  relational_database_name = "scc-db-1"
+  availability_zone        = "ap-northeast-2a"
+  master_database_name     = "scc"
+  master_password          = data.sops_file.secret_data.data["db.password"]
+  master_username          = "scc"
+  blueprint_id             = "postgres_12"
+  bundle_id                = "micro_2_0"
+  skip_final_snapshot      = true
+}
```
Comment (Zeniuus) [inline infra/terraform/lightsail/lightsail_rdb.tf:10]: 생각해보니 DB는 그냥 한 대만 띄우고, 테썹 / 실섭 schema를 다르게 해서 ㄱㄱ?

**Review by Zeniuus — APPROVED** [review]
리베이스만 부탁드립니당

File: infra/terraform/lightsail/secret.yaml (hunk 2 above)
Comment (bellatoris) [reply infra/terraform/lightsail/secret.yaml:10]: 네 맞아요

File: infra/terraform/lightsail/lightsail_rdb.tf (hunk 3 above)
Comment (bellatoris) [reply infra/terraform/lightsail/lightsail_rdb.tf:10]: 그렇게 가면 되지 않을가요

**Comment by bellatoris** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Stair-Crusher-Club/scc-server/76?utm_source=stack-comment).


Job Reference: 4MceVfTtllLb7fOU89IP

**Pull request merged**